### PR TITLE
Add vehicle parts to the tool quality screen

### PR DESCRIPTION
**Pull request by CalKerethi**
Quick change to the quality screen, adding a new section for vehicle parts where applicable.

### CODE REVIEW

**Comment by CalKerethi** [comment]
Amended the original commit to clean up a missing newline at the end.

File: src/types/ToolQuality.svelte [hunk 1]
```diff
@@ -18,6 +18,17 @@ for (const it of data.byType('item')) {
     toolsWithQualityByLevel.get(level).push(it)
   }
 }
+
+let vpartsWithQualityByLevel = new Map<number, VehiclePart[]>()
```
Comment (nornagon) [inline src/types/ToolQuality.svelte:22]:
```suggestion
const vpartsWithQualityByLevel = new Map<number, VehiclePart[]>()
```

File: src/types/ToolQuality.svelte [hunk 2]
```diff
@@ -18,6 +18,17 @@ for (const it of data.byType('item')) {
...
+for (const it of data.byType('vehicle_part')) {
```
Comment (nornagon) [inline src/types/ToolQuality.svelte:23]:
```suggestion
for (const it of data.byType<VehiclePart>('vehicle_part')) {
```

**Review by nornagon — APPROVED** [review]

**Comment by nornagon** [comment]
This is live now :)

**Pull request merged**